src/Apps/CollectorProfile/Routes/Saves/Components/EditListPrivacyModal/EditArtworkListItem.tsx

```diff
@@ -0,0 +1,120 @@
+import { Flex, Image, Toggle, Text, Spacer, THEME } from "@artsy/palette"
+import { FC } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { extractNodes } from "Utils/extractNodes"
+import { EditArtworkListItem_item$data } from "__generated__/EditArtworkListItem_item.graphql"
+import { useFormikContext } from "formik"
+import { __internal__useMatchMedia } from "Utils/Hooks/useMatchMedia"
+import NoArtIcon from "@artsy/icons/NoArtIcon"
+import { EditListPrivacyFormModel } from "Apps/CollectorProfile/Routes/Saves/Components/EditListPrivacyModal/EditListPrivacyModal"
+import HideIcon from "@artsy/icons/HideIcon"
+
+interface EditArtworkListItemProps {
+  item: EditArtworkListItem_item$data
+}
+
+const EditArtworkListItem: FC<EditArtworkListItemProps> = props => {
+  const { item } = props
+  const artworkNodes = extractNodes(item.artworksConnection)
+  const imageURL = artworkNodes[0]?.image?.resized?.src ?? null
+  const totalArtworks = item.artworksCount ?? 0
+  const { values, setFieldValue } = useFormikContext<EditListPrivacyFormModel>()
+  const isMobile = __internal__useMatchMedia(THEME.mediaQueries.xs)
```

Comment:
Note that this works here because we're only rendering the modal on the client, but if this were to ever intersect the SSR path it would error out. Best to avoid it as its only for framework-level stuff. 

A better way to do this would be to lean on our responsive props, or our Media helper:

1) 
```tsx
<Media at='xs'> 
  <Text variant="sm" display>
    {values[item.internalID] ? "Shared" : "Private"}
  </Text>
</Media>
```

2)

```tsx
<Text variant="sm" display={['none', 'block']}>
  {values[item.internalID] ? "Shared" : "Private"}
</Text>
````

Reply:
Also, just for FYI, a third way to get mobile info is `getENV('IS_MOBILE')`, which is set at the express level during the SSR pass.